src/controllers/UsersController.php

```diff
@@ -386,6 +386,38 @@ public function deleteUser($request, $db)
         $view->setResponseCode(204);
     }
 
+    /**
+     * Allow users to be set as trusted
+     *
+     * @param $request Request
```

Comment:
These are type hinted here but not in the function itself. Can they be added there too please